recipes/psiclass/meta.yaml

```diff
@@ -0,0 +1,37 @@
+package:
+  name: psiclass
+  version: 1.0.2
+
+source:
+  url: https://github.com/splicebox/PsiCLASS/archive/v1.0.2.tar.gz
+  sha256: 3958e38e071d02c239fb8748f213fe96c724c06860fbc381cdeba5331bafa6ae
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+  host:
+    - samtools 0.1.19.*
+    - zlib
+  run:
+    - zlib
+    - perl
+
+test:
+  commands:
+    - classes 2>&1 | cat > /dev/null
+    - junc 2>&1 | cat > /dev/null
+    - trust-splice 2>&1 | cat > /dev/null
+    - subexon-info 2>&1 | cat > /dev/null
+    - combine-subexons 2>&1 | cat > /dev/null
+    - vote-transcripts 2>&1 | cat > /dev/null 
+    - add-genename 2>&1 | cat > /dev/null
+    - which psiclass
+
+about:
+  home: https://github.com/splicebox/PsiCLASS
+  license: GPLv3
```

Comment:
Can you pleae use this identifier: https://spdx.org/licenses/GPL-3.0.html and also include `licnese:`

Reply:
I changed the license to "license: GNU General Public License v3.0 only". Is it fine? Thank you!